modules/gapi/src/backends/common/serialization.hpp

```diff
@@ -130,6 +130,9 @@ GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::gapi::wip::draw::Poly
 GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::gapi::wip::draw::Line &l);
 GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::gapi::wip::draw::Line &l);
 
+GAPI_EXPORTS IOStream& operator<< (IOStream& os, const cv::gapi::ot::TrackingStatus& status);
+GAPI_EXPORTS IIStream& operator>> (IIStream& is,       cv::gapi::ot::TrackingStatus& status);
```

Comment:
Is it required now for status? I believe s11n can be omitted for the tracking status.

Reply:
Fixed!